### PR TITLE
Mat 3507 getting measure from path

### PR DESCRIPTION
**Pull request by gregory-akins**
## MADiE PR

Jira Ticket: [MAT-3507](https://jira.cms.gov/browse/MAT-3507)
(Optional) Related Tickets:

### All Submissions
* [X] This PR has the JIRA linked.
* [X] Required tests are included.
* [X] No extemporaneous files are included (i.e Complied files or testing results).
* [X] This PR is merging into the **correct branch**.
* [X] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [X] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/types/measure.d.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export interface Measure {
+  id: {
+    timestamp: number;
+    date: string;
+  };
```
Comment (jkotanchik-SB) [inline src/types/measure.d.ts:5]: Should this `id` also be updated to `string`?

File: src/types/measure.d.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  measureHumanReadableId: string;
+  measureSetId: string;
+  version: number;
+  revisionNumber: number;
+  state: string;
+  name: string;
```
Comment (jkotanchik-SB) [inline src/types/measure.d.ts:11]: `name` was recently changed to `measureName`.

Comment (RohitKandimalla) [reply src/types/measure.d.ts:11]: I don't think we require this, since we have a new "models" directory to handle typescript types.

**Review by jkotanchik-SB — APPROVED** [review]

**Review by jkotanchik-SB — DISMISSED** [review]

**Review by AndrewBird81 — DISMISSED** [review]

File: src/types/measure.d.ts (hunk 2 above)
Comment (gregory-akins) [reply src/types/measure.d.ts]: I just removed it and updated the PR

**Review by RohitKandimalla — APPROVED** [review]

**Pull request merged**